README.rst

```diff
@@ -0,0 +1,301 @@
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| APL (Apollo Lake)    | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| KBLx [1]             | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| ICL (Ice Lake)       | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| JSL (Jasper Lake)    | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| EHL (Elkhart Lake)   | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| TGL (Tiger Lake)     | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| DG1 (Xe MAX)         | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| SG1                  | Legacy       | ✔               |                 | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| RKL (Rocket Lake)    | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| ADL-S (Alder Lake S) | Legacy       | ✔               | ✔               | ✔               |                 |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| ADL-P (Alder Lake P) | VPL          |                 | ✔               |                 | ✔               |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| Future platforms...  | VPL          |                 | ✔               |                 | ✔               |
++----------------------+--------------+-----------------+-----------------+-----------------+-----------------+
+| Multi GPU            | Legacy + VPL | See above                         | ✔               | via env var [2] |
```

Comment:
Dispatcher loads VPL in multi GPU case and using MSDK_RUNTIME environment variable is not implemented yet.